aspects/aspects.go

```diff
@@ -650,30 +707,83 @@ func NewJSONDataBag() JSONDataBag {
 // Get takes a path and a pointer to a variable into which the value referenced
 // by the path is written. The path can be dotted. For each dot a JSON object
 // is expected to exist (e.g., "a.b" is mapped to {"a": {"b": <value>}}).
-func (s JSONDataBag) Get(path string, value interface{}) error {
+func (s JSONDataBag) Get(path string, value *interface{}) error {
 	subKeys := strings.Split(path, ".")
 	return get(subKeys, 0, s, value)
 }
 
-func get(subKeys []string, index int, node map[string]json.RawMessage, result interface{}) error {
+func get(subKeys []string, index int, node map[string]json.RawMessage, result *interface{}) error {
 	key := subKeys[index]
+	matchAll := key[0] == '{' && key[len(key)-1] == '}'
```

Comment:
would a isPlaceholder helper be useful? another approach would be to use dedicated types/interface for subKeys instead of just string...

Reply:
We have that helper but I forgot to use it. Corrected this here and in other places I had missed. Thanks